src/Modules/Profie/index.js

```diff
@@ -72,6 +73,37 @@ class Profile extends Component {
         });
       });
   };
+  handleSubmitEditProfile = () => {
+    const { bio, username, data: userData } = this.state;
+    const data = new FormData();
+    data.append('bio', bio);
+    data.append('username', username);
+    axios({
+      method: 'patch',
+      url: 'https://calm-waters-47062.herokuapp.com/users',
+      headers: {
+        Authorization: `Bearer ${localStorage.getItem('cb-token')}`,
+        'Content-Type': 'multipart/form-data',
+      },
+      data,
+    }).then((res) => {
+      if (res.data.success) {
+        localStorage.setItem('cb-username', username);
```

Comment:
Are you sure, nothing else needs to be updated in the localstorage?

Reply:
Yes, because we are saving token, email and username only to localStorage, and the updated thing here is only the username from the local storage